static/components.js

```diff
@@ -108,6 +108,27 @@ define(function () {
                 }
             };
         },
+        getRtlView: function() {
+            return {
+                type: 'component',
+                componentName: 'rtl',
+                componentState: {}
+            };
+        },
+        getRtlViewWith: function (id, source, rtlOutput, compilerName, editorid) {
+            return {
+                type: 'component',
+                componentName: 'rtl',
+                componentState: {
+                    id: id,
+                    source: source,
```

Comment:
What is the source needed for on the view? I cant find any usage for it on the code right noe